scripts/install_third_party_libs.py

```diff
@@ -242,6 +253,61 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
```

Comment:
Can you add tests for the new code you've added here? I see that you've updated the tests file, but it doesn't look like those comprehensively test the newly-added code